data/datasets.json

```diff
@@ -1,4 +1,22 @@
 {
+    "boss": {
+        "Connectomics_Department": {
+            "ara": {
+                "domain": "https://api.boss.neurodata.io",
+                "collection": "ara_2016",
+                "experiment": "sagittal_50um",
+                "username": "jstriebel",
+                "password": "{,vEzT7J?-5_"
+            },
+            "kasthuri2011-neurodata-io": {
+                "domain": "https://api.boss.neurodata.io",
+                "collection": "kasthuri",
+                "experiment": "kasthuri11",
+                "username": "jstriebel",
```

Comment:
@normanrz I would like to check in one or two datasets, so that there is immediately something to try out. Could you create a user on neurodata.io with scmboy for that?

Reply:
or @hotzenklotz? ^

Reply:
Hmm. I think it is reasonable to ask people to create an account when they want to use the boss integration. I wouldn't want to put credentials into the repo.

Reply:
When using it with wk, one will be asked via the frontend anyways. This one would be rather for development, where I would find it useful if we have one or two demo-datasets in the repo, that are working out-of-the box (for dev & review).
The credentials can not be used for anything, except retrieving data from neurodata.io and possibly exceeding a rate-limit, if there is one.

Reply:
I could also read credentials from an gitignored `.env` file as a fallback. That would make it easier to develop locally without caring about checking in personal credentials by accident or not.

Reply:
I implemented the second option now (env vars), this seems quite useful.